hack/00-osd-managed-cluster-config.selectorsyncset.yaml.tmpl

```diff
@@ -4722,8 +4727,8 @@ objects:
     - apiVersion: user.openshift.io/v1
       kind: Group
       metadata:
-        name: osd-sre-cluster-admins
-      users: []
+        name: osd-sre-admins
+      users: ${{SREP_USERS}}
```

Comment:
`SREP_USERS` needs to be defined in the template.  Similar to this:
https://github.com/openshift/managed-cluster-config/blob/master/scripts/templates/template.yaml#L31-L32

Reply:
thanks for catching this! I've added it in.